<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionp</class>
 <widget class="QWidget" name="gestionp">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="gp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>491</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Gestionproduits.png</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>681</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter un produit</string>
    </attribute>
    <widget class="QLabel" name="ap">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>681</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../back.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="remise">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>270</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="quantite">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QLineEdit" name="ref">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prix">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>100</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: transparent;
color:white;</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>180</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: transparent;
color:white;
</string>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QComboBox" name="categorie">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="fournisseur">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="oui">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>oui</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="non">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>non</string>
     </property>
    </widget>
    <widget class="QLabel" name="ok">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ann">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>170</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#186838;&quot;&gt;Reference :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#186838;&quot;&gt;Nom:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#186838;&quot;&gt;Fournisseur:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#186838;&quot;&gt;Categorie:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#186838;&quot;&gt;Quantite:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#186838;&quot;&gt;Prix:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#186838;&quot;&gt;Remise :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>ap</zorder>
    <zorder>remise</zorder>
    <zorder>nom</zorder>
    <zorder>quantite</zorder>
    <zorder>ref</zorder>
    <zorder>prix</zorder>
    <zorder>categorie</zorder>
    <zorder>fournisseur</zorder>
    <zorder>oui</zorder>
    <zorder>non</zorder>
    <zorder>ok</zorder>
    <zorder>pushButton</zorder>
    <zorder>ann</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>label_7</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Consulter produits</string>
    </attribute>
    <widget class="QLabel" name="cp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>681</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../back.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: transparent;
color:white;</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: transparent;
color:white;
</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>200</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: transparent;
color:white;</string>
     </property>
     <property name="text">
      <string>Noter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>270</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: transparent;
color:white;</string>
     </property>
     <property name="text">
      <string>Commenter</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="refe">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rem">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="notee">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pri">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="com">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>290</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ref_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ref:</string>
     </property>
    </widget>
    <widget class="QLabel" name="n">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
    <widget class="QLabel" name="c">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Categorie:</string>
     </property>
    </widget>
    <widget class="QLabel" name="f">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fournisseur:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pr">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>prix:</string>
     </property>
    </widget>
    <widget class="QLabel" name="r">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>remise:</string>
     </property>
    </widget>
    <widget class="QLabel" name="q">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantite:</string>
     </property>
    </widget>
    <widget class="QLabel" name="co">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>290</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Commentaire:</string>
     </property>
    </widget>
    <widget class="QLabel" name="no">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Note:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="qu">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="oui_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>oui</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="non_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>non</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="four">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cat">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="refr">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="refrech">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:transparent;
color:white;</string>
     </property>
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="mod">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="suppr">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>130</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="nt">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>190</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="commen">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>260</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../bouton.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>cp</zorder>
    <zorder>tableView</zorder>
    <zorder>refe</zorder>
    <zorder>rem</zorder>
    <zorder>nom1</zorder>
    <zorder>notee</zorder>
    <zorder>pri</zorder>
    <zorder>com</zorder>
    <zorder>ref_2</zorder>
    <zorder>n</zorder>
    <zorder>c</zorder>
    <zorder>f</zorder>
    <zorder>pr</zorder>
    <zorder>r</zorder>
    <zorder>q</zorder>
    <zorder>co</zorder>
    <zorder>no</zorder>
    <zorder>qu</zorder>
    <zorder>oui_2</zorder>
    <zorder>non_2</zorder>
    <zorder>four</zorder>
    <zorder>cat</zorder>
    <zorder>refr</zorder>
    <zorder>refrech</zorder>
    <zorder>mod</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>suppr</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>nt</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>commen</zorder>
    <zorder>pushButton_6</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border : trasnaprent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="rech">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:transparent;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recherche">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>20</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="foo">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="caa">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="fo">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ca">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
